migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc

```diff
@@ -0,0 +1,266 @@
+* [ ] Disable Tracing: `spec.tracing.type=None`
+** [ ] Configure the `ServiceMeshControlPlane` with OpenShift Distributed Tracing as the replacement. link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-configuring-distr-tracing-tempo_observability[Configuring Distributed Tracing Tempo]
+* [ ] Disable Kiali: `spec.addons.kiali.enabled=false`
+** [ ] Create a standalone `Kiali` resource as the replacement. If you haven't already done this as part of the previous items for Prometheus or Tracing, follow link:https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0.0tp1/html/observability/kiali-operator-provided-by-red-hat#ossm-install-kiali-operator_ossm-kiali-assembly[Kiali Operator provided by Red Hat] on how to install and configure Kiali as a standalone resource.
+
+== Checklist for migrating {SMProduct} 2 routing and gateways
+
+This section applies if you have not already moved to explicitly-managed routes and gateway injection.
+
+Remember that automatic route creation, also known as Istio OpenShift Routing (IOR), is a deprecated feature that is disabled by default for any `ServiceMeshControlPlane` resource that was created using {SMProduct} 2.5 and later. To move from {SMProduct} 2 to {SMProduct} 3, you need to migrate from IOR to explicitly-managed routes.
+
+* [ ] Migrate from Istio OpenShift Routing (IOR) to to explicitly-managed routes. link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-route-migration[Service Mesh Route Migration]
+
+Since the {SMProduct} 3 control plane does not manage gateways, you must manage them yourself. To do so, and move from {SMProduct} 2 to {SMProduct} 3, you need to disable the default gateways, move away from the Service Mesh Control Plane declaration, and begin to use gateway injection.
+
+* [ ] Migrate to gateway injection link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-gateway-migration[Service Mesh Gateway Migration] to migrate off of the default ingress/egress gateways.
+
+== Checklist for disabling network policy management and Grafana in {SMProduct} 2
+
+  * [ ] Disable network policy management: `spec.security.manageNetworkPolicy=false`.
+
+If you cannot disable network policies and go without them for the duration of the migration for security reasons, see: <insert exref>[Migrating network policies for security purposes]
+
+Otherwise, if you want to keep the Network Policies created by the {SMProduct} 2.6 `ServiceMeshControlPlane`, you will need to recreate and manage these manually. See <exreft>[migrating network policies]
```

Comment:
"If you cannot disable network policies" sentence and "if you want to keep the Network Policies" sentence means the same thing so "Otherwise"  is confusing here.